src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -123,52 +123,16 @@ protected override void ProcessRecord()
 
                     break;
             }
-
+            
             foreach (string path in pathsToProcess)
             {
                 byte[] bytehash = null;
                 string hash = null;
-                Stream openfilestream = null;
-
-                try
+                
+                if (ComputeFileHash(path, out hash))
```

Comment:
Please use modern syntax:
```c#
                if (ComputeFileHash(path, out string hash))
```

Reply:
The term "modern syntax" does not mean anything to me - which part is not modern, and what is the modern version?

Reply:
C# 8.0 allow inline variable definition. So we can remove `string hash = null;` and use `if (ComputeFileHash(path, out string hash))`

Reply:
I understand, thank you; code is updated.